en/Obsidian Publish/Redirecting old notes.md

```diff
@@ -0,0 +1,23 @@
+Renaming and removing notes is a natural part of maintaining a living vault. While Obsidian automatically updates links when you move a note within your local vault, other websites may still link to your old notes on your published [[Introduction to Obsidian Publish|Obsidian Publish]] site. In this article, you'll learn how to redirect readers from one note to another.
+
+Imagine you want to move a note from one folder to another:
+
+- **Guides**
+  - ~~Making friends.md~~ (removed)
+- **Tutorials**
+  - *How to make friends.md* (added)
+
+After you move the note, Obsidian automatically updates all links within the vault. However, if you publish the change to your Publish site, any links to `/Guides/Making+friends` will result in a 404.
+
+To redirect readers from `/Guides/Making+friends` to `/Tutorials/How+to+make+friends`, you need to add an [[Aliases|alias]] in `How to make friends.md`, the note you want to redirect to.
+
+```md
+---
+alias: Guides/Making friends
+---
```

Comment:
This is brilliant - thank you.

I wonder, could there be a second example that says what to do if you rename the file again - that is, what to make it look like if you want to add a second redirect?

Or format the first one so that it will work if a second line is added....

Reply:
Great point! I'll update the example.